zscaler_connector.py

```diff
@@ -822,6 +823,169 @@ def _handle_get_admin_users(self, param):
 
         return action_result.set_status(phantom.APP_SUCCESS)
 
+    def _handle_get_users(self, param):
+        """
+        This action is used to fetch all users
+        :param name: User name
+        :param dept: User department
+        :param group: User group
+        :param limit: Max number of users to retrieve
+        :return: status phantom.APP_ERROR/phantom.APP_SUCCESS(along with appropriate message)
+        """
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+
+        # Add an action result object to self (BaseConnector) to represent the action for this param
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        if not param:
+            return action_result.set_status(phantom.APP_ERROR, "No filters provided")
+
+        ret_val, limit = self._validate_integer(action_result, param.get('limit', ZSCALER_MAX_PAGESIZE), ZSCALER_LIMIT_KEY)
+        if phantom.is_fail(ret_val):
+            return action_result.get_status()
+        params = {
+            "name": param.get('name'),
+            "dept": param.get('dept'),
+            "group": param.get('group'),
+            'page': 1
+        }
+        users = []
+        while True:
+            if limit < ZSCALER_MAX_PAGESIZE:
+                params['pageSize'] = limit
+            else:
+                params['pageSize'] = ZSCALER_MAX_PAGESIZE
+            ret_val, get_users = self._make_rest_call_helper('/api/v1/users', action_result, params=params)
+            if phantom.is_fail(ret_val):
+                return action_result.get_status()
+            for user in get_users:
+                users.append(user)
```

Comment:
suggestion:
What do you think about this variable names?
get_users => users or current_users
users => all_users